articles/active-directory/develop/active-directory-claims-mapping.md

```diff
@@ -191,7 +191,7 @@ There are certain sets of claims that define how and when they're used in tokens
 |`http://schemas.microsoft.com/identity/claims/identityprovider`|
 |`http://schemas.microsoft.com/identity/claims/objectidentifier`|
 |`http://schemas.microsoft.com/identity/claims/puid`|
-|`http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier [MR1] `|
```

Comment:
Looked at the history and couldn't see where the `[MR1]` was coming from, and it isn't part of the schema url

Reply:
Thanks @nschonni